src/js/components/Chart.js

```diff
@@ -582,6 +582,25 @@ export default class Chart extends Component {
     return (pos1 && pos2 && pos1.endX > pos2.startX && pos1.startX < pos2.endX);
   }
 
+  _splitLabel (label, position, labelY) {
```

Comment:
Actually var labelArray = label.split(','); should be changed to var labelArray = label.split(delimiter); 
It'll be invoked like this:
<Chart series={series} delimiter="," points={true} legend={{position: 'overlay'}} type="line" xAxis={xAxis} />

This is particularly useful for users who have date as labels for x axis.  

Creating a label string with a \n in it will not help as it'll still have one text element and we need tspans for split.
Here is how it looks after adding \n

![image](https://cloud.githubusercontent.com/assets/17637919/13897423/4d8e5d1a-edd7-11e5-8462-bdabd8e6cdf2.png)